### ISSUE TITLE
list downstream packages incompatible with new breaking release?

### ISSUE THREAD

**Issue by ericphanson** [JuliaRegistries/RegistryCI.jl#296]
We could use @KristofferC's https://github.com/KristofferC/RegistryCompatTools.jl to add a spoiler-tagged list of packages that would hold back the new breaking release to a comment in the registry PR. That way, the author could see it and decide if they want to rethink making a breaking release or want to help some of the downstream repos.

Perhaps there could even be a prompt for a list of breaking changes for the release notes or something like that (with some kind of tagbot integration).

**Comment by DilumAluthge** [comment]
Would this be a suggestion, or would it actually block AutoMerge?

**Comment by ericphanson** [comment]
I think it shouldn’t block, at least as automerge works now where only a maintainer can unblock it.

Maybe it could be a new kind of check where the author can unblock it? I think skipping a version would fit as that kind of check too. For both it’s ok as long as the author knows, and it gives them a chance to fix it. Maybe it could even be unblocked by a thumb up reaction from the author or something really simple like that.

**Comment by ericphanson** [comment]
Closing in favor of #375. Sorry, probably I should’ve written that as a comment here instead, but now it is more up to date with what I think we should do so I’ll keep it open and close this one.
